src/OpenLoco/src/OpenLoco.cpp

```diff
@@ -168,11 +168,6 @@ namespace OpenLoco
         call(0x004058F5);
     }
 
-    static void sub_4062E0()
-    {
-        call(0x004062E0); // getTime unused Dead code
-    }
-
     static bool sub_4034FC(int32_t& a, int32_t& b)
     {
         auto result = ((int32_t(*)(int32_t&, int32_t&))(0x004034FC))(a, b);
```

Comment:
Looking at 0x004034FC (`ddraw_lock_psurface`), I think this can also be removed at this point.

Reply:
Oh yeah there are much more to remove this we just a couple